azext_iot/operations/_mqtt.py

```diff
@@ -109,18 +109,11 @@ def twin_patch_handler(self, patch):
             self.device_client.patch_twin_reported_properties(modified_properties)
 
     def execute(self, data, properties={}, publish_delay=2, msg_count=100):
-        from azext_iot.operations.hub import _iot_device_twin_update
         from tqdm import tqdm
 
         try:
             if self.init_reported_properties:
-                twin_properties = {
-                    "properties": {
-                        "desired": self.init_reported_properties
-                    }
-                }
-
-                _iot_device_twin_update(self.target, self.device_id, twin_properties)
+                self.device_client.patch_twin_reported_properties(self.init_reported_properties)
```

Comment:
Are there IT's for this functionality? If not, we need them.

Reply:
Yes, the integration test already exists.